pom.xml

```diff
@@ -20,11 +20,22 @@
   <name>GitHub API Plugin</name>
   <url>https://wiki.jenkins-ci.org/display/JENKINS/GitHub+API+Plugin</url>
 
+  <properties>
+    <jenkins.version>1.615</jenkins.version>
```

Comment:
Given that `okhttp` needs Java 7, let's bump baseline to the first version using Java 7.